### PR TITLE
More granular module tests

### PR DESCRIPTION
**Pull request by murrant**
Hopefully improve readability of failures
Open to suggestions.

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
Hoping this resolves the issues with truncated outputs in laravel6 branch... worth testing

File: tests/OSModulesTest.php [hunk 1]
```diff
@@ -113,39 +113,42 @@ public function testOS($os, $variant, $modules)
         $debug = in_array('--debug', $_SERVER['argv'], true);
```
Comment (Jellyfrog) [inline tests/OSModulesTest.php:113]: Listen to Scrutinizer

**Comment by Jellyfrog** [comment]
Do we have some exempel of the output?

**Comment by murrant** [comment]
No, I need to do more testing and maybe a little more refinement here.

**Comment by murrant** [comment]
to be revisited.

**Pull request closed without merging**